ui/src/AppSharing/AppSharing.svelte

```diff
@@ -53,22 +53,31 @@
 </script>
 
 <div class="sharing">
-  <button on:click={() => (visible = true)}>Embed and Share Info</button>
+  <button on:click={() => (visible = true)} on:keypress={() => (visible = true)}
```

Comment:
not sure if this is the desired behavior, but you may want to narrow the key that triggers this event to something like "Enter"

https://www.w3schools.com/howto/howto_js_trigger_button_enter.asp

Reply:
I'm pretty sure this one was from svelte-check asking for an "on:keypress" to go along with the "on:click" so I just copied the handler. Limiting it to "Enter" makes sense.